backend/api/Areas/Admin/Mapping/AccessRequest/AccessRequestMap.cs

```diff
@@ -11,7 +12,7 @@ public void Register(TypeAdapterConfig config)
             config.NewConfig<Entity.AccessRequest, Model.AccessRequestModel>()
                 .Map(dest => dest.Id, src => src.Id)
                 .Map(dest => dest.Status, src => src.Status)
-                .Map(dest => dest.Agencies, src => src.AgenciesManyToMany)
+                .Map(dest => dest.Agencies, src => src.AgenciesManyToMany.OrderBy(a => a.Agency.ParentId))
```

Comment:
The frontend displays the first agency if the backend returns more then one. This order by ensures that agencies with null parent ids (ie. parent agencies) are listed first. This prevents a bug where if an agency is changed on the frontend to a parent agency, the frontend would display one of that parent agencies children instead.